utils_nlp/models/transformers/named_entity_recognition.py

```diff
@@ -398,7 +405,9 @@ def fit(
 
         # init scheduler
         scheduler = Transformer.get_default_scheduler(
-            optimizer=self.optimizer, warmup_steps=warmup_steps, num_training_steps=max_steps
+            optimizer=self.optimizer,
```

Comment:
should we use dataset instead of dataloader to make it consistent with other transformer models?

Reply:
same for prediction

Reply:
can the predict function take the label_map to provide the entity labels directly without having the users call the get_predicted_token_labels?